JavaScript/JavaScriptSDK/JavaScriptSDK.csproj

```diff
@@ -56,6 +56,15 @@
     <TypeScriptRemoveComments>false</TypeScriptRemoveComments>
     <TypeScriptSourceMap>true</TypeScriptSourceMap>
     <TypeScriptModuleKind>AMD</TypeScriptModuleKind>
+    <TypeScriptJSXEmit>None</TypeScriptJSXEmit>
+    <TypeScriptCompileOnSaveEnabled>True</TypeScriptCompileOnSaveEnabled>
+    <TypeScriptNoImplicitAny>False</TypeScriptNoImplicitAny>
+    <TypeScriptOutFile>min\$(OutFileBaseName).js</TypeScriptOutFile>
+    <TypeScriptOutDir />
+    <TypeScriptNoEmitOnError>True</TypeScriptNoEmitOnError>
+    <TypeScriptMapRoot>
+    </TypeScriptMapRoot>
+    <TypeScriptSourceRoot>https://raw.githubusercontent.com/Microsoft/ApplicationInsights-JS/master/JavaScript</TypeScriptSourceRoot>
```

Comment:
If my understanding is correct, this will cause source map to point at the current state of master, which may be out of date? Should we instead be pointing at sources with release tag? Would this require making this variable set at build time?